bidrag-transport/src/main/kotlin/no/nav/bidrag/transport/behandling/felles/grunnlag/Særbidrag.kt

```diff
@@ -0,0 +1,33 @@
+@file:Suppress("ktlint:standard:filename")
+
+package no.nav.bidrag.transport.behandling.felles.grunnlag
+
+import io.swagger.v3.oas.annotations.media.Schema
+import no.nav.bidrag.domene.enums.særbidrag.SærbidragKategori
+import no.nav.bidrag.domene.enums.særbidrag.Utgiftstype
+import java.math.BigDecimal
+import java.time.LocalDate
+
+data class SærbidragKategoriGrunnlag(
+    val kategori: SærbidragKategori,
+    @get:Schema(
+        description =
+            "Beskrivelse på kategorien. Er påkrevd hvis ANNET er satt",
+    )
+    val beskrivelse: String? = null,
+) : GrunnlagInnhold
+
+data class SærbidragUtgiftspostGrunnlag(
```

Comment:
Særbidragsutgiftspostsgrunnlag evnt bare Utgiftspostsgrunnlag med mindre vi kjenner til andre behandlingstyper som utgift vil være relevant for?